pull-request/src/person.js

```diff
@@ -11,5 +11,10 @@ Person.prototype.fullname = function() {
 }
 
 Person.prototype.wishHappyBirthday = function() {
+    if (this.age === NaN) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Use a test of the format "a === a" instead. ([javascript:S2688](http://localhost:9750/coding_rules#rule_key=javascript%3AS2688))

[See it in SonarQube](http://localhost:9750/project/issues?id=training%3Apull-request&pullRequest=35&issues=AWgTXqWA6PH_c9BW2L54&open=AWgTXqWA6PH_c9BW2L54)

[ISSUE_KEY]: AWgTXqWA6PH_c9BW2L54